web/ui/dashboard/src/app/app-routing.module.ts

```diff
@@ -24,6 +27,7 @@ const routes: Routes = [
 	},
 	{
 		path: '',
+		resolve: [() => getSignupConfig()],
```

Comment:
I don't think this is the right place for this. Actually, I don't think we should have it at all.

This guy only has business with the login and signup page, let those two pages handle it, along with a loader state.

Reply:
I added this here to be able to get the signup config value whenever a public route is being accessed, calling this only at the login page would restrict fetching the signup config value to only when the login page is being accessed, we can't fetch this value on the signup page as this page shouldn't even be accessed at all if the config value is being returned as `false`, hence why this is here, on any instance, a public route is always the first route accessed before any other activity is carried out, hence why that function is being called there, so the signup guard would have an accurate value to work with at all times because the config has already been fetched.

Reply:
The signup guard does this exact same thing